hack/push-release.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script pushes all of the built images to a registry.
+#
+# Set OS_PUSH_BASE_REGISTRY to prefix the destination images
```

Comment:
I understand you're copying these scripts from elsewhere. Maybe it would be useful to check that `OS_PUSH_BASE_REGISTRY` is set early in the script execution?

Apparently if it is not set the script still attempts to push the image to Docker Hub (empty prefix)?!